src/test/java/org/broadinstitute/hellbender/engine/FeatureManagerUnitTest.java

```diff
@@ -33,7 +32,6 @@
                 { new File(FEATURE_MANAGER_TEST_DIRECTORY + "minimal_vcf3_file.vcf"), VCF3Codec.class },
                 { new File(FEATURE_MANAGER_TEST_DIRECTORY + "minimal_bcf_file.bcf"), BCF2Codec.class },
                 { new File(FEATURE_MANAGER_TEST_DIRECTORY + "minimal_bed_file.bed"), BEDCodec.class},
-                { new File(publicTestDir + "org/broadinstitute/hellbender/utils/codecs/hapmap/genotypes_chr1_ASW_phase3.3_first500.hapmap"), RawHapMapCodec.class },
               // TODO: Table codec (and other "reference-dependent" codecs) currently broken. Need to
               // TODO: refactor them so that they don't require a GenomeLocParser
               //  { new File(FEATURE_MANAGER_TEST_DIRECTORY + "minimal_table_file.table"), TableCodec.class}
```

Comment:
done, and I reinstated the test
`{ new File(FEATURE_MANAGER_TEST_DIRECTORY + "minimal_table_file.table"), TableCodec.class}`